### PR TITLE
Added new frontend project Iphone 14 pro clone

### PR DESCRIPTION
**Pull request by vansh-codes**
Closes: #433 

- **Title** : IPHONE 14 Pro Clone
- **Name:** : Vansh Chaurasiya
- **Idenitfy yourself:** : GSSoC Contributor

### Describe the add-ons or changes you've made 📃
Updated the databaes frontend.json file and added my project tags and description. Updated the frontend folder readme and added my project's link also.
Added files for my project along with a webp screenshot of the clone website.

### How Has This Been Tested? ⚙️

Tested locally

### Screenshots 📷

image

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Front-end-Projects/Intermediate/IPHONE-14-Pro-Clone/Screenshot.webp
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/IPHONE-14-Pro-Clone/Screenshot.webp:1]: rename it to `screenshot`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@vansh-codes resolve merge conflicts

File: Front-end-Projects/Intermediate/IPHONE-14-Pro-Clone/Screenshot.webp
Comment (vansh-codes) [reply Front-end-Projects/Intermediate/IPHONE-14-Pro-Clone/Screenshot.webp:1]: Resolved. Please check now

**Review by Avdhesh-Varshney — COMMENTED** [review]
Re-raise your by updating your forked repository.

**Pull request closed without merging**